.editorconfig

```diff
@@ -561,4 +561,410 @@ dotnet_diagnostic.CS0618.severity = suggestion
 # Missing XML comment for publicly visible type or member '...'
 dotnet_diagnostic.CS1591.severity = suggestion
 
+# SA1600: Elements should be documented
+dotnet_diagnostic.SA1600.severity = suggestion
```

Comment:
All of these changes appear to be duplicated above. Can you revert all this and selectively disable only new warnings that were introduced with .NET 6?

Reply:
done. removed all the duplicates in [this commit](https://github.com/microsoft/component-detection/pull/288/commits/fa8fce6651bb6d7fa8e5efa23707563b8b4ecb70)